projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/recommendations/Hero.svelte

```diff
@@ -26,8 +26,11 @@
 		<span>Latest report as of {lastUpdated}</span>
 		{#if $imageDataActiveGroup.issue_count}
 			<h1>
-				{$imageDataActiveGroup.issue_count}
-				Image Recommendations
+				{sprintf(
+					/* translators: %d: number of image recommendations */
+					__( '%d Image Recommendations', 'jetpack-boost' ),
```

Comment:
Since this is a header, I think we should use `_n` and be more accurate.

```suggestion
					_n( '%d Image Recommendation', '%d Image Recommendations', $imageDataActiveGroup.issue_count, 'jetpack-boost' ),
```